js/coinex.js

```diff
@@ -1285,6 +1286,71 @@ module.exports = class coinex extends Exchange {
         return this.parseTrades (trades, market, since, limit);
     }
 
+    async fetchFundingHistory (symbol = undefined, since = undefined, limit = 100, params = {}) {
```

Comment:
For the future, let's keep the signature uniform where possible, and set the default value for the limit inside this method.